content/en/guides/models/app/features/panels/_index.md

```diff
@@ -195,7 +195,11 @@ To add a new section before or after an existing section, you can instead click
 
 
 ### Manage a section's panels
-Sections with a large number of panels are paginated by default if they use the **Standard grid** layout. The default number of panels on a page depend on the panel's configuration and on the sizes of the panels in the section.
+Sections with a large number of panels are paginated by default. The default number of panels on a page depend on the panel's configuration and on the sizes of the panels in the section.
+
+{{% alert %}}
+The **Custom grid** layout will soon be removed. For that reason, you can no longer update a workspace to use it, but you can manually update a workspace that uses **Custom grid** to **Standard grid**. When the **Custom grid** layout is removed, workspaces will be updated to use the default layout.
```

Comment:
```suggestion
The **Custom grid** layout will soon be removed. W&B suggests that you no longer use Custom grid layouts. Consider updating your workspace from **Custom grid** to **Standard grid**.

When the **Custom grid** layout is removed, workspaces will be updated to use the **Standard grid** layout.
```

Unsure about the last sentence, is standard grid the default layout? If nay, forget the last sentence.